scrapers/sis_scraper/prerequisites.py

```diff
@@ -55,10 +56,14 @@ def parse_atom(tokens: Tokens, cur_tok: int):
         result, cur_tok = parse_tokens(tokens, cur_tok + 1)
         assert tokens[cur_tok][0] == "CLOSE_PAREN"
     else:
+        min_grade_exists = tokens[cur_tok + 1][0] == "MIN_GRADE"
```

Comment:
What if this is the last token?

Reply:
Good point­– this did not happen in the entire data set but I should add a check for that.

Reply:
Jk, there's the END token

Reply:
Nvm looks like it's actually impossible for that to happen due to END existing